common/djangoapps/util/views.py

```diff
@@ -308,7 +322,7 @@ def _record_feedback_in_zendesk(
 
 
 def _record_feedback_in_datadog(tags):
-    datadog_tags = [u"{k}:{v}".format(k=k, v=v) for k, v in tags.items()]
+    datadog_tags = [u"{k}:{v}".format(k=k, v=v) for k, v in sorted(tags.items())]
```

Comment:
We're sorting them because `tags` is a dictionary and therefore order is not guaranteed when calling `tags.items()`. This can result in flaky tests, as the tests are looking for a specific request to be sent to datadog with a list of tags in a certain order.